charts/incubator/onlyoffice-ds-ce/Chart.yaml

```
@@ -0,0 +1,22 @@
+apiVersion: v2
+appVersion: "6.4.1.45"
+dependencies:
+- name: common
+  repository: https://truecharts.org
+  version: 8.0.13
+description: "ONLYOFFICE Document Server is an online office suite comprising viewers and editors for texts, spreadsheets and presentations, fully compatible with Office Open XML formats: .docx, .xlsx, .pptx and enabling collaborative editing in real time."
+home: https://github.com/truecharts/apps/tree/master/charts/stable/onlyoffice-ds-ce
+icon: https://avatars.githubusercontent.com/u/1426033?s=200&v=4
+keywords:
+- onlyoffice
+kubeVersion: '>=1.16.0-0'
+maintainers:
+- email: info@truecharts.org
+  name: TrueCharts
+  url: truecharts.org
+name: onlyoffice-ds-ce
```

This is also whats displayed in the GUI. DS CE seems super weird.
Please also rename the folder accordingly.

```suggestion
name: Onlyoffice-Document-Server
```